lib/pipelineFactory.js

```diff
@@ -49,9 +51,16 @@ class PipelineFactory extends BaseFactory {
 
         modelConfig.createTime = (new Date()).toISOString();
 
-        return userFactory.get({ username: Object.keys(config.admins)[0] })
+        return userFactory.get({
+            username: Object.keys(config.admins)[0],
+            scmContext: config.scmContext
+        })
```

Comment:
The indentation of this block looks funny. The closing brackets should line up with the `.then` on the next line.